### DISCUSSION TITLE
Layout recommendation

### DISCUSSION THREAD

**Discussion by jaymehta-twilio** [twilio-labs/paste#1087] [Q&A] [answered]
Hello team, what is a recommended way to layout the stack buttons as shown, aligned left and right respectively ?
<img width="574" alt="Screen Shot 2021-01-11 at 2 47 37 PM" src="https://user-images.githubusercontent.com/52717786/104248318-1f8cb800-541e-11eb-93db-950bd4969869.png">

**Comment by SiTaggart** [comment]
Hi  @jaymehta1986, just to clarify, is this a design question; "How should I place these buttons in this card?" or is this an engineering question; "How would I build this design?"

**Reply by jaymehta-twilio** [reply]
This was more about building this in the code. What components should I use ideally as I don't find a direct way to use align them either side. I'm still familiarizing myself with the Paste components as there're many newly added ones. :)

**Comment by SiTaggart** [comment, marked as answer]
A very quick way to achieve this in code might be to use the Flex component which implements the Flexbox css property. https://paste.twilio.design/layout/flex

```
<Flex hAlignContent="between" vAlignContent="center">
  ...
</Flex>
```

You can also just use the Box component and apply CSS props that way:  https://paste.twilio.design/primitives/box

**Reply by jaymehta-twilio** [reply]
Thanks @SiTaggart.. I think Flex seems a clean solution here and it just works.
